launcher/config.go

```diff
@@ -241,8 +242,18 @@ func configurePropagators(c *LauncherConfig) error {
 }
 
 func newResource(c *LauncherConfig) *resource.Resource {
+	// workaround until the following change is released
+	// https://github.com/open-telemetry/opentelemetry-go/pull/1042
+	reset := false
+	if len(os.Getenv("OTEL_RESOURCE_LABELS")) == 0 {
```

Comment:
put this environment variable name in a constant?

Reply:
i thought about it, but figured the code will be removed shortly anyways

Reply:
👍 SG